sdk/core/http/azure-pipelines.yml

```diff
@@ -0,0 +1,83 @@
+trigger:
+- master
```

Comment:
I assume this is another file that should be deleted as part of the clean-up.

Reply:
Yep, I'll remove this and any other CI or GitHub repo-specific files that aren't needed post-migration